.github/workflows/build-image.yml

```diff
@@ -0,0 +1,75 @@
+# Manually triggered using GitHub UI.
+
+name: Publish image to Registry
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.1
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
+          version: v1.32
+
+          # Optional: working directory, useful for monorepos
+          # working-directory:
+
+          args: --timeout 10m
+          # Optional: show only new issues if it's a pull request. The default value is `false`.
+          # only-new-issues: true
+
+  unit-test:
+    name: Unit tests for Atlas-operator
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.1
+
+      - name: Unit test
+        run: |
+          echo "Unit test"
+
+  build:
+    name: Push to Registry
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2.3.1
+
+    - name: Push Atlas-operator to Local Registry
+      id: registry
+      run: |
+        image="localhost:5000/dev-test:latest"
```

Comment:
1. push to dockerhub ("prerelease")
2. as discussed: for tag name: "branch-commid_id_short"